pod/favorite/templates/favorite/favorite_video_list.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+{% load static %}
+{% spaceless %}
+<div class="pod-infinite-container infinite-container" id="videos_list" nextPage = "{{ videos.has_next }}" data-countvideos="{{ count_videos }}">
```

Comment:
pourquoi ne pas utiliser le templates video/templates/videos/video_list.html ?

Reply:
Dans le `favorite_video_list.html`, il y a l'ajout d'une `div` et la modification d'une autre pour pouvoir faire le drag and drop. Le script JS n'est pas le même.